packages/devtools_app/lib/src/screens/network/network_service.dart

```diff
@@ -44,7 +48,10 @@ class NetworkService {
   /// time.
```

Comment:
please update dart doc references to NetworkController.lastHttpDataRefreshTime now that that does not exist